cargo-pgx/src/command/install.rs

```diff
@@ -146,13 +152,18 @@ pub(crate) fn install_extension(
             "control file",
             true,
             &package_manifest_path,
+            versioned_so,
         )?;
     }
 
     {
         let mut dest = base_directory.clone();
         dest.push(&pkgdir);
-        dest.push(format!("{}.so", extname));
+        if versioned_so {
+            dest.push(format!("{}-{}.so", extname, extver));
```

Comment:
this incantation also exists over in `pgx_sql.rs`.  How can we centralize them?

Reply:
I don't think that there is any value in trying to do that. There are similarities, yes, but by the time you would have refactored everything to pull out the commonality, the common piece would be this:

```
pub fn build_versioned_so_name(extname: String, extver: String) -> String {
    format!("{}-{}", extname, extver)
}
```

Reply:
I see. I've fixed this in the most recent iteration.